BUILDING.md

```diff
@@ -171,10 +273,3 @@ cmake -S openassetio -B build --install-prefix ~/dist \
 
 Then we can run the usual steps (see above) to build and install
 OpenAssetIO, and run the tests (if enabled).
-
-## CMake presets
```

Comment:
[question] Any reason not to mention the presets any more? Maybe they could do with the options as its very closely related - ie; so people aren't bamboozled by the options list?

Reply:
Hmm, not sure why we originally decided to omit them, i guess they seemed a little unmoored. Nonetheless I love the idea of framing them as extentions to the cmake options section, [cbec5fa](https://github.com/OpenAssetIO/OpenAssetIO/pull/711/commits/cbec5fa797f4bdb8417b3cf9d4f4ea59454061bd)